app/policies/standup_meeting_group_policy.rb

```diff
@@ -11,6 +11,10 @@ def create?
     user.admin?
   end
 
+  def new?
+    user.admin?
+  end
```

Comment:
We don't need this because of some inheritance stuff happening. If you look in `ApplicationPolicy` (which this class inherits from), you'll see that `new?` looks at what's happening with `create?`, which hopefully makes sense as those actions are closely related.

Given `create?` is defined to look for `user.admin`, you can just remove this and have it inherit the parent's definition for `new?`